### ISSUE TITLE
Kiosk mode not working in web apps

### ISSUE THREAD

**Issue by manjit24** [filips123/PWAsForFirefox#176]
it would be great if we can start WebApps directly in kiosk mode please consider this in future updates or if there is a way to do it currently, HELP.

**Comment by filips123** [comment]
What do you mean with kiosk mode?

**Comment by manjit24** [comment]
if we pass -Kiosk parameter in the shortcut of Firefox with URL we will launch a site in locked full screen window for just one site like mentioned here  ->
https://support.mozilla.org/en-US/kb/firefox-enterprise-kiosk-mode
likewise if we do with PWA it would be great.

**Comment by filips123** [comment]
I currently cannot test this, but I think it should work if you pass `-kiosk` argument to Firefox when launching PWA from command line: `firefoxpwa site launch PWA-ID -- -kiosk` (`--` is important)

To get the PWA ID, run `firefoxpwa profile list`. ID is printed inside parentheses for each PWA.

**Comment by manjit24** [comment]
i tried this is not working ...it only works when no other instance of Firefox is running.

**Comment by filips123** [comment]
Closing this issue because this is how normal Firefox also works. If you find that something behaves differently than normal Firefox, please comment and I will reopen it.

**Comment by crashbandicode** [comment]
Hello I'd like to revive this issue. It seems that the behavior is different than normal firefox. The pwa launches maximized but not fullscreen whereas the firefox kiosk launches in fullscreen mode. Perhaps there's a different way I'm supposed to be passing the arguments? 

I've attached a recording of the behavior. I tested this on two different machines and the behavior is the same so it should be easy to reproduce


https://github.com/user-attachments/assets/44e6d250-3c11-488c-8374-6e48447631b1

Note if you push f11 in the pwa it goes to full screen but it should start in full screen.
